testing/infra/terraform/modules/standalone_apm_server/variables.tf

```diff
@@ -33,50 +43,20 @@ variable "stack_version" {
   type        = string
 }
 
-variable "region" {
-  default     = "gcp-us-west2"
-  description = "Optional ESS region where to run the smoke tests"
-  type        = string
-}
-
-variable "worker_region" {
-  default     = "us-west-2"
-  description = "Optional AWS region where the workers will be created. Defaults to us-west-2 (AWS)"
-  type        = string
-}
-
 variable "ea_managed" {
   default     = false
   description = "Whether or not install Elastic Agent managed APM Server"
   type        = bool
 }
 
+variable "apm_server_bin_path" {
+  default     = ""
+  type        = string
+  description = "Optionally use the apm-server binary from the specified path instead"
+}
+
 variable "tags" {
   type        = map(string)
   default     = {}
   description = "Optional set of tags to use for all deployments"
 }
-
-# CI variables
```

Comment:
What's the reason for deleting these labels?

Reply:
This module didn't use those variables, so I decided to replace it unified `tags` variable instead.